tests/seeding/test_find_or_create.py

```diff
@@ -63,6 +63,25 @@ def list(self, page: Optional[int] = None, per_page: int = 100):
             else:
                 return self.projects[(page - 1)*per_page:page*per_page]
 
+        def search(self, search_params: Optional[dict] = None, per_page: int = 100):
```

Comment:
unused?

Reply:
find_collection calls `search` now, so this is used (and w/o it the tests fail).